cli/utilities/packages.py

```diff
@@ -211,13 +212,96 @@ def unregister(self):
             )
 
     def status(self):
+        """
+        Get the current subscription status.
+
+        Returns:
+            str: Subscription status output.
+        """
         cmd = f"{self.base_cmd} status"
 
         out = self.execute(sudo=True, cmd=cmd)
         if isinstance(out, tuple):
             return out[0].strip()
         return out
 
+    def clean(self):
+        """
+        Cleans all local subscription data: entitlements, certs, identity.
+
+        Raises:
+            SubscriptionManagerError: if clean fails.
+        """
+        self.execute(sudo=True, long_running=False, cmd=f"{self.base_cmd} clean")
+
+    def cleanup_subscription_files(self):
+        """
+        Remove residual subscription-related certificate and identity files.
+        This method removes files that are not cleaned by `subscription-manager clean`,
+        such as consumer and product certificates.
+        Raises:
+            SubscriptionManagerError: if the file cleanup command fails.
+        """
+        cmd = "rm -rf /etc/pki/consumer /etc/pki/product"
+        self.execute(sudo=True, long_running=False, cmd=cmd)
+
+    @retry(CommandFailed, tries=10, delay=30)
+    def enable_rhel_rpms(ceph, distro_ver):
+        """
+        Enable required RHEL repositories based on the distro version.
+
+        Args:
+            ceph: The node or context to run commands on.
+            distro_ver (str): The major RHEL version (e.g., '7', '8', '9')
+        """
+        repos = {
+            "7": ["rhel-7-server-rpms", "rhel-7-server-extras-rpms"],
+            "8": ["rhel-8-for-x86_64-appstream-rpms", "rhel-8-for-x86_64-baseos-rpms"],
+            "9": ["rhel-9-for-x86_64-appstream-rpms", "rhel-9-for-x86_64-baseos-rpms"],
+        }
+
+        ceph.exec_command(
+            sudo=True, cmd=f"subscription-manager release --set {distro_ver}"
+        )
+        for repo in repos.get(distro_ver[0]):
+            ceph.exec_command(
+                sudo=True,
+                cmd=f"subscription-manager repos --enable={repo}",
+                long_running=True,
+            )
+
+    @retry(CommandFailed, tries=10, delay=30)
+    def enable_rhel_eus_rpms(ceph, distro_ver):
+        """
+        Enable Extended Update Support (EUS) repositories for RHEL 7.
+
+        Args:
+            ceph: The node or object to run commands on.
+            distro_ver (str): The major RHEL version (e.g., '7').
+
+        Raises:
+            NotImplementedError: If the version is not supported for EUS.
+        """
+        eus_repos = {"7": ["rhel-7-server-eus-rpms", "rhel-7-server-extras-rpms"]}
+        for repo in eus_repos.get(distro_ver[0]):
+            ceph.exec_command(
+                sudo=True,
+                cmd=f"subscription-manager repos --enable={repo}",
```

Comment:
Use - https://github.com/red-hat-storage/cephci/blob/main/cli/utilities/packages.py#L230

Reply:
Sure, I’ll update it to use the existing function as suggested.